resources/swagger.json

```diff
@@ -55,6 +55,49 @@
         }
       }
     },
+    "/users/search": {
+      "get": {
+        "summary": "Search users by filters",
+        "operationId": "searchUser",
+        "security": [
+          {
+            "OauthSecurity": [
+              "management"
+            ]
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/User"
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/responses/Standard401ErrorMessage"
+          },
+          "403": {
+            "$ref": "#/responses/Standard403ErrorMessage"
+          },
+          "404": {
```

Comment:
404 is not appropriate for this operation, since we are not passing a specific user id. 
If there are no users found for the search criteria, then it will return 200 with an empty list